recipes-devtools/clang/clang.inc

```diff
@@ -7,7 +7,10 @@ LLVM_GIT_PROTOCOL ?= "git"
 #
 # Uncomment below to enable master version of clang/llvm
 #
-BASEPV = "4.0.0"
+MAJOR_VER="4"
+MINOR_VER="0"
+PATCH_VER="0"
+BASEPV = "${MAJOR_VER}.${MINOR_VER}.${PATCH_VER}"
```

Comment:
May be we dont need BASEPV now. Just use the MAJOR MINOR PATCH to achieve the same goal throughout.